### ISSUE TITLE
Remove warning when quitting Ganga after running GUI

### ISSUE THREAD

**Issue by egede** [ganga-devs/ganga#1948]
When quitting Ganga after running the GUI, the warning message
```
/home/egede/programming/ganga/ganga/ganga/GangaGUI/api/routes.py:1096: UserWarning: The 'environ['werkzeug.server.shutdown']' function is deprecated and will be removed in Werkzeug 2.1.
  func()
```
pops up. This should be removed.

**Comment by gurudattapatil** [comment]
I searched about the error current version of werkzeug is 2.0.1, and this says that it will be solved in 2.1 version.

**Comment by egede** [comment]
OK, no worries then.
